examples/plot_new_metrics.py

```diff
@@ -109,7 +109,11 @@ def marriage_transform(m_s_string):
 def occupation_transform(occ_string):
     """Perform some simple manipulations."""
     result = "Small"
-    if occ_string.startswith("Machine"):
+    if isinstance(occ_string, float):
+        # Had missing data
+        assert np.isnan(occ_string)
+        result = "Low"
```

Comment:
Should this say "Small"?

Reply:
Yes. I must have read the wrong `fillna()` value below

Reply:
I'll update the logic to correct this.